### PR TITLE
Use consistent image

### PR DESCRIPTION
**Pull request by metanerd**

### CODE REVIEW

**Comment by metanerd** [comment]
Breaking build step is known to the introducer: https://github.com/mattermost/mmctl/commits/master
I think this could be merged already, even before the build is fixed.

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ commands:
     parameters:
       version:
         type: string
-        default: 1.31.0
+        default: 1.42.0
```
Comment (streamer45) [inline .circleci/config.yml:28]: Is this needed or just updating while we are here?

Comment (metanerd) [reply .circleci/config.yml:28]: Just updating while we are here.

Comment (streamer45) [reply .circleci/config.yml:28]: Alright, as long as it doesn't cause new errors we should be good :)

**Review by streamer45 — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (metanerd) [reply .circleci/config.yml:28]: You are right, might need a cache key bump.

**Pull request merged**